server/routes/findInterventions/searchResultsView.ts

```diff
@@ -5,14 +5,62 @@ import { SummaryListArgs, SummaryListItem } from '../../utils/summaryList'
 export default class SearchResultsView {
   constructor(private readonly presenter: SearchResultsPresenter) {}
 
+  private checkboxArgs({
+    name,
+    legend,
+    checkboxes,
+  }: {
+    name: string
```

Comment:
Probably one to add to the `govuk-frontend-types` that we introduced in a recent PR if we're keen to go down the route of strong types here, but we've got that covered in [this jira issue](https://dsdmoj.atlassian.net/browse/IC-1173?atlOrigin=eyJpIjoiODgwMDczYzBhZjQ1NGY5YWFhMmNjZjVmODdhYmM1MzUiLCJwIjoiaiJ9)

Reply:
Good point! I'll leave it until we work on that.